app/assets/stylesheets/styles/game.scss

```diff
@@ -71,6 +71,11 @@
   width: 50px;
   height: 50px;
 }
+
+.gamelist {
+  color: #9bb0bb;
```

Comment:
Color literals like `#9bb0bb` should only be used in variable declarations; they should be referred to via variable everywhere else.